packages/address/src/loader.ts

```diff
@@ -18,7 +18,9 @@ const HEADERS = {
   'Access-Control-Allow-Origin': '*',
 };
 
-export async function loadCountries(locale: string): Promise<Country[]> {
+export const loadCountries: (
+  locale: string,
+) => Promise<Country[]> = memoizeAsync(async (locale: string) => {
```

Comment:
Do you need an explicit return type?

Reply:
@clauderic says it's nice to have one in case I change my function's signature 👅